### DISCUSSION TITLE
Commits not showing in dashboard

### DISCUSSION THREAD

**Discussion by IvanROU** [gogs/gogs#7247] [General]
I have imported a lot of repositories into the Gogs. This was done manually by editing the DB (sqlite).

However, when I commit some changes and push them to the repository nothing appears in the Dashboard. When I enter the repository in Gogs web panel everything is fine, commit history appears and so on. How can this be fixed?

**Comment by IvanROU** [comment]
We eventually found the source of the problem. It turned out to have access rights to the gogs-repositories folder. 
They were installed in 755, and after changing to 775 everything worked correctly.

**Reply by marcelpfutz** [reply]
I have the same problem.
I've granted the rights to 775, but it still doesn't update the dashboard.
What tables is it based on?
